api/src/main/kotlin/edu/wgu/osmt/elasticsearch/SearchController.kt

```diff
@@ -4,24 +4,37 @@ import edu.wgu.osmt.PaginationDefaults
 import edu.wgu.osmt.RoutePaths
 import edu.wgu.osmt.api.GeneralApiException
 import edu.wgu.osmt.api.model.*
+import edu.wgu.osmt.collection.Collection
 import edu.wgu.osmt.collection.CollectionDoc
 import edu.wgu.osmt.collection.CollectionEsRepo
 import edu.wgu.osmt.config.AppConfig
+import edu.wgu.osmt.config.SEARCH_BY_API_THRESHOLD
 import edu.wgu.osmt.db.PublishStatus
 import edu.wgu.osmt.jobcode.JobCodeEsRepo
 import edu.wgu.osmt.keyword.KeywordEsRepo
 import edu.wgu.osmt.keyword.KeywordTypeEnum
-import edu.wgu.osmt.richskill.RichSkillDoc
-import edu.wgu.osmt.richskill.RichSkillEsRepo
+import edu.wgu.osmt.richskill.*
```

Comment:
Would you mind please setting your editor to not reduce imports to wildcards? It's fine for trivial Java imports, but it's helpful to see what each class needs by these declarations.

- https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Reply:
gotcha